cms/djangoapps/cms_user_tasks/tasks.py

```diff
@@ -26,6 +26,11 @@ def send_task_complete_email(self, task_name, task_state_text, dest_addr, detail
         'DISABLE_CMS_TASK_EMAILS',
         settings.FEATURES.get('DISABLE_CMS_TASK_EMAILS', True)
     )
+    if disable_emails == 'true':
```

Comment:
Can we write this statement in single line?

Reply:
Done